### PR TITLE
MGMT-7108 Adds labels to the new validations

### PR DESCRIPTION
**Pull request by jkilzi**
- Updates swagger.yaml (in sync with assisted-service)
- Adds labels to the following new validations:
  -  'sufficient-network-latency-requirement-for-role'
  - 'sufficient-packet-loss-requirement-for-role'
  - 'has-default-route'

### CODE REVIEW

File: src/api/swagger.yaml [hunk 1]
```diff
@@ -113,6 +111,14 @@ paths:
           type: string
           format: uuid
           required: false
+        - in: query
+          name: ams_subscription_ids
```
Comment (mareklibra) [inline src/api/swagger.yaml:115]: nice, we can continue with MGMT-6468

File: src/api/swagger.yaml [hunk 2]
```diff
@@ -3019,6 +3027,13 @@ paths:
         - userAuth: [admin, read-only-admin, user]
       description: Get minimum host requirements.
       operationId: GetHostRequirements
+      parameters:
```
Comment (mareklibra) [inline src/api/swagger.yaml:3030]: We can enhance UI for that. Are SNO HW requirements in the UI tracked somewhere? @jelkosz

File: src/api/types.ts [hunk 3]
```diff
@@ -1594,6 +1600,10 @@ export interface VersionedHostRequirements {
    * Worker node requirements
    */
   worker?: ClusterHostRequirementsDetails;
+  /**
+   * Single node OpenShift node requirements
+   */
+  sno?: ClusterHostRequirementsDetails;
```
Comment (mareklibra) [inline src/api/types.ts:1606]: We can enhance UI for SNO HW reqs.

**Review by mareklibra — APPROVED** [review]

File: src/api/types.ts (hunk 3 above)
Comment (jkilzi) [reply src/api/types.ts:1606]: Agree, task here [MGMT-7135](https://issues.redhat.com/browse/MGMT-7135)

File: src/api/swagger.yaml (hunk 2 above)
Comment (jkilzi) [reply src/api/swagger.yaml:3030]: https://issues.redhat.com/browse/MGMT-7135

**Pull request merged**